_maps/RandomRuins/LavaRuins/lavaland_surface_dorffort.dmm

```diff
@@ -0,0 +1,57 @@
+"a" = (/turf/template_noop,/area/template_noop)
+"b" = (/turf/closed/mineral/ash_rock,/area/mine/explored)
```

Comment:
Please make it use normal lavaland rock and basalt floors instead of these sprites.